web/src/components/insight-page/chart-base/index.tsx

```diff
@@ -95,7 +97,8 @@ export const ChartBase: FC<ChartBaseProps> = ({
           type: "value",
           name: yName,
           nameLocation: "center",
-          nameGap: 32,
+          nameGap: 50,
+          max: yMax,
```

Comment:
should add yMax to dependencies list of this useEffect as well.

Reply:
Nice catch! Fixed it.